web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -475,17 +495,58 @@ class SpeakPage extends React.Component<Props, State> {
 
   private agreeToTerms = async () => {
     this.setState({ showPrivacyModal: false });
+    this.setState({ privacyAgreedChecked: true });
     this.props.updateUser({ privacyAgreed: true });
     this.upload(true);
   };
 
+  private onPrivacyAgreedChange = (privacyAgreed: boolean) => {
+    this.setState({ privacyAgreedChecked: privacyAgreed });
+  };
+
   private toggleDiscardModal = () => {
     this.setState({
       showPrivacyModal: false,
       showDiscardModal: !this.state.showDiscardModal,
     });
   };
 
+  private handleSubmit = (evt: React.SyntheticEvent) => {
+    const { user } = this.props;
+
+    const hasSeenFirstCTA = window.sessionStorage.getItem(SEEN_FIRST_CTA);
```

Comment:
How about something like ```
// utils.js
const castTrueString = (strValue: string) => strValue === 'true' ? true : false;

const hasSeenFirstCTA = castTrueString(window.sessionStorage.getItem(SEEN_FIRST_CTA));

```

then in the conditions below we don't need to check the actual string value since its a boolean.

Reply:
Yeah that's a really good suggestion 👌 as an improvement I think we can drop the ternary expression because the value returned will always be `false` if the string passed in is not equal to the string true and `true` if it is.